doc/source/contributing.rst

```diff
@@ -814,9 +816,8 @@ We would name this file ``test_cool_feature.py`` and put in an appropriate place
 .. code-block:: python
 
    import pytest
-   import numpy as np
-   import pandas as pd
-   from pandas.util import testing as tm
+   import numpy as np        # noqa: F811
+   import pandas as pd       # noqa: F811
```

Comment:
same

Reply:
This similar to the issue above. The tutorial presents a complete block of code to be copied and saved as a .py file. Without the explicit import, anyone following the tutorial will not be able to replicate the results. It might make sense to include this block as a non executable block.